### PR TITLE
Upgrade dependencies to match quality checklist

### LINKED ISSUE #63: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

**This checklist is used internally but can be used as a guidance for others**

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] `LICENSE` file exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [ ] Packages/dependencies are up to date

- [ ] Code compiles against latest packages/dependencies

- [ ] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [ ] Functionality tests have been created

- [ ] All tests pass

### App

- [ ] Confirmed sample behaves as expected (after dependencies have been updated)

- [ ] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [ ] App is released under a permissive license like MIT or Apache-2

- [ ] I've read and followed the contribution guidelines

- [ ] I've read and implemented my template according to the respective coding guidelines for my project

- [ ] I agree and adhere to the Code of Conduct

- [ ] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by rojasTob**
Resolves #63

### CODE REVIEW

File: app/assets/config/manifest.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```
Comment (po5i) [inline app/assets/config/manifest.js:1]: Is this file required?

Comment (rojasTob) [reply app/assets/config/manifest.js:1]: It is needed by the Sprockets gem (Rails asset pipeline is implemented by this gem).
